dgorin1/mddtoolkit
# Formatting v2

# How to test
for mac:
install miniconda if you haven't yet: https://docs.conda.io/projects/conda/en/latest/user-guide/install/macos.html
Create a new environment with `conda create -n diffusion_modeller python=3.11`
Cd to the root of this project and run the command `./setup.sh`
While in the root of this project, run the command `python src/diffusion_kinetics/main.py` to make sure it runs
